src/test/rgw/bucket_notification/api.py

```diff
@@ -247,13 +247,15 @@ def delete_all_topics(conn, tenant, cluster):
     if tenant == '':
         topics_result = admin(['topic', 'list'], cluster)
         topics_json = json.loads(topics_result[0])
-        for topic in topics_json['topics']:
+        topics = topics_json.get('topics', [])
```

Comment:
but in this case we do not clear v1 topics.
maybe better to check if `topics` is empty, and try to: `topics_json.get('result, [])` ?
if the value here is not empty, get the `topics`.

in this case, it might be simpler to use the `KeyError` exception